Poker.BE/Poker.BE.Service/Modules/Requests/SignUpRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Poker.BE.Service.Modules.Requests
+{
+	public class SignUpRequest : IRequest
+	{
+		/// <summary>
+		/// requesting user hash code
+		/// </summary>
+		public string UserName { get; set; }
```

Comment:
❗️  its a `string` user name? or a user address in-memory (hash code)? those a two different things! please advise. @galwainer and fix if needed.